### DISCUSSION TITLE
Match file names before file paths in file picker

### DISCUSSION THREAD

**Discussion by keirlawson** [helix-editor/helix#7774] [Ideas]
when searching for my `Main.scala` file in a scala project the picker instead matches every single file in `/src/main` when I search for `main`.  VS Code by contrast will match `Main.scala` first in this case, and I believe Helix should follow this behavior.

**Comment by pascalkuthe** [comment]
Can you lost a screenshot. The `Main.scala` file should have a higher fuzzy score than `src/main/too.scala` and should be shown first. In general there is no special logic for files in the licker. All pickers use a generic fuzzy matching algorithm so there will be no slecical handeling for the filename

**Comment by keirlawson** [comment]
I suspect the issue is I search all in lower case, so it matches the path first as the file name is capitalised.  Has there been any discussion of having the file picker only search file names, rather than file names + paths? This could be something I could look at contributing if its desirable.
